src/ansys_sphinx_theme/layout.html

```diff
@@ -3,6 +3,8 @@
 {# Append our custom CSS after the bootstrap stylesheet so we can override where necessary #}
 {%- block css %}
     {{ super() }}
+    {# adding support for plotly in gallery examples #}
```

Comment:
Can we hide this behind a feature flag in the `conf.py`? It's adding 3.3MB to the page size, and presumably eats some performance on page load to execute the JS.

Reply:
that's a very valid point.

I could create a flag in the `conf.py` file, so it will load it only if activated. Would it work for you?

(btw, who checks the size of the packages?? hahaah #Joking)

Reply:
Thought the same. This basically requires `plotly` for each and every page.

I think the best approach is to run a "cleanup" script that simply swaps the position of the two lines while submitting an issue (and ideally PR) to the sphinx-gallery team (or applicable repo) to clean up the actual issue.

Reply:
Why not something like:

```
{# adding support for plotly in gallery examples #}
    {% if theme_use_plotly %}
        <script src="https://cdn.plot.ly/plotly-latest.min.js"></script>
    {% endif %}
```

and then add:
```py
use_plotly=True
```
to the `theme.conf` file???

Reply:
But I agree about filing a PR to a higher package. I wonder if `pydata-sphinx-theme` is better objective.

Reply:
@akaszynski is your concern that this still loads on every page of the documentation (if enabled)? In terms of size I think that's less of a concern (after it's loaded once it should be cached?), but still needs to execute the JS (take that with a grain of salt, I'm not an expert on these things).

Of course, if we can fix it "upstream" that would be the ideal solution.

Reply:
> Why not something like:
> 
> ```
> {# adding support for plotly in gallery examples #}
>     {% if theme_use_plotly %}
>         <script src="https://cdn.plot.ly/plotly-latest.min.js"></script>
>     {% endif %}
> ```
> 
> and then add:
> 
> ```python
> use_plotly=True
> ```
> 
> to the `theme.conf` file???

That will still require that each and every page executes the plotly JS regardless of if it's used. Still feels like a bad solution relative to simply patching the HTML until it's fixed upstream.

Reply:
I'm not expert either... @akaszynski  should I take the time to propose something in `ipywidgets` repo?

Reply:
Try first with the `sphinx-gallery`